Scarb.toml

```diff
@@ -7,7 +7,8 @@ cairo-version = "2.6.3"
 [dependencies]
 cubit = { git = "https://github.com/influenceth/cubit.git", commit = "62756082bf2555d7ab25c69d9c7bc30574ff1ce8" }
 starknet = ">=1.3.0"
-snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.19.0" }
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.20.1" }
```

Comment:
Yeahhh... I had issues running the repo using the older version of foundry and after upgrading the foundry version i was able to run the previous test implementations without any errors i was having previously. Might have just been a dependency issue on my end but this actually fixed it, someone else might also experience this so i thought it might be nice to avoid such issues in future.